docs/en_US/reference/experiment_config.rst

```diff
@@ -309,11 +321,11 @@ use active gpu
 
 Specify whether NNI should submit trials to GPUs occupied by other tasks.
 
-type: ``bool``
+type: ``Optional[bool]``
 
-If your are using desktop system with GUI, set this to ``True``.
+Must be set when `trial gpu number` greater than zero.
```

Comment:
Do this field has default value?

Reply:
No. Default to `False` is problematic for local mode, and default to `True` is reversing previous behavior.